examples/conf/vlcs_diva_mldg_dial.yaml

```diff
@@ -5,7 +5,9 @@ val_threshold: 0.8                  # threashold before which training does not
 model: dann_diva                    # combine model DANN with DIVA
 epos: 1                             # number of epochs
 trainer: mldg_dial                  # combine trainer MLDG and DIAL
-gamma_reg: 1.0                      # hyperparameter of DANN
+gamma_reg: 
+  default: 1.0
```

Comment:
do you think it is more appropriate to name "default" to be "reg_trainer"? @MatteoWohlrapp

Reply:
i understand now, so we can use default .

Reply:
No because as mentioned above, trainers can also be specified individually.